modules/operator-roles/README.md

```diff
@@ -2,19 +2,19 @@
 
 ## Introduction
 
-This Terraform sub-module is responsible for overseeing the management of IAM roles utilized by operators within the cluster for their necessary actions within the AWS account.
+This Terraform sub-module manages the IAM roles used by operators within the cluster for their necessary actions in the AWS account.
 
-Included in this sub-module are the following permissions:
+The following permissions are included in this sub-module:
 - ROSA Ingress Operator IAM role.
 - ROSA back-end storage IAM role.
 - ROSA Machine Config Operator role.
 - ROSA Cloud Credential Operator role.
 - ROSA Image Registry Operator role.
 
-For more info see:
+For more information, see:
```

Comment:
If it isn't a huge ask, could we restructure this as a single sentence with the links?